src/libsync/discovery.h

```diff
@@ -298,7 +299,7 @@ class ProcessDirectoryJob : public QObject
 
     FolderQuota _folderQuota;
 
-    int64_t folderQuotaAvailable(const SyncFileItemPtr &item);
+    int64_t folderBytesAvailable(const SyncFileItemPtr &item, const bool isServerEntryValid);
```

Comment:
replace `bool` by an enum type to have a better API (way more readable than `false` or `true`)